### PR TITLE
[DOCS] Adds description to Node APIs

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

Related to https://github.com/elastic/elasticsearch-specification/issues/964.

Adds descriptions to Node API properties. Source of info: 
* https://www.elastic.co/guide/en/elasticsearch/reference/master/cluster.html

### CODE REVIEW

File: specification/nodes/_types/Stats.ts [hunk 1]
```diff
@@ -112,45 +246,133 @@ export class SerializedClusterStateDetail {
...
+  /**
+   * The cumulative amount of time spent computing no-op cluster state updates since the node started.
+   */
   computation_time?: Duration
+  /**
+   * The cumulative amount of time, in milliseconds, spent computing no-op cluster state updates since the node started.
+   */
   computation_time_millis?: DurationValue<UnitMillis>
+  /**
+   * The cumulative amount of time spent publishing cluster state updates which ultimately succeeded, which includes everything from the start of the publication (i.e. just after the computation of the new cluster state) until the publication has finished and the master node is ready to start processing the next state update.
```
Comment (abdonpijpelink) [inline specification/nodes/_types/Stats.ts:292]:
```suggestion
   * The cumulative amount of time spent publishing cluster state updates which ultimately succeeded, which includes everything from the start of the publication (just after the computation of the new cluster state) until the publication has finished and the master node is ready to start processing the next state update.
```

File: specification/nodes/_types/Stats.ts [hunk 2]
```diff
@@ -112,45 +246,133 @@ export class SerializedClusterStateDetail {
...
+   * This includes the time measured by `context_construction_time`, `commit_time`, `completion_time` and `master_apply_time`.
+   */
   publication_time?: Duration
+  /**
+   * The cumulative amount of time, in milliseconds, spent publishing cluster state updates which ultimately succeeded, which includes everything from the start of the publication (i.e. just after the computation of the new cluster state) until the publication has finished and the master node is ready to start processing the next state update.
```
Comment (abdonpijpelink) [inline specification/nodes/_types/Stats.ts:297]:
```suggestion
   * The cumulative amount of time, in milliseconds, spent publishing cluster state updates which ultimately succeeded, which includes everything from the start of the publication (just after the computation of the new cluster state) until the publication has finished and the master node is ready to start processing the next state update.
```

File: specification/nodes/_types/Stats.ts [hunk 3]
```diff
@@ -160,58 +382,157 @@ export class KeyedProcessor {
...
+  /**
+   * The exponentially weighted moving average service time of search requests on the keyed node.
+   */
   avg_service_time?: Duration
+  /**
+   * The exponentially weighted moving average service time, in nanoseconds, of search requests on the keyed node.
+   */
   avg_service_time_ns?: long
+  /**
+   * The number of outstanding search requests from the node these stats are for to the keyed node.
```
Comment (abdonpijpelink) [inline specification/nodes/_types/Stats.ts:425]:
```suggestion
   * The number of outstanding search requests to the keyed node from the node these stats are for.
```

Comment (abdonpijpelink) [reply specification/nodes/_types/Stats.ts:425]: I had to read this sentence five times to understand what it says. Maybe this makes it easier to read?

File: specification/nodes/_types/Stats.ts [hunk 4]
```diff
@@ -160,58 +382,157 @@ export class KeyedProcessor {
...
   overhead?: float
+  /**
+   * Total number of times the circuit breaker has been triggered and prevented an out of memory error.
+   */
   tripped?: float
 }
 
 export class Cgroup {
+  /**
+   * Contains statistics about cpuacct control group for the node.
```
Comment (abdonpijpelink) [inline specification/nodes/_types/Stats.ts:463]:
```suggestion
   * Contains statistics about `cpuacct` control group for the node.
```

File: specification/nodes/_types/Stats.ts [hunk 5]
```diff
@@ -236,137 +563,382 @@ export class DataPathStats {
...
+  /**
+   * Total amount of unallocated disk space in the file store.
+   */
   free?: string
+  /**
+   * Total number of unallocated bytes in the file store.
+   */
   free_in_bytes?: long
+  /**
+   * Mount point of the file store (ex: /dev/sda2).
```
Comment (abdonpijpelink) [inline specification/nodes/_types/Stats.ts:575]:
```suggestion
   * Mount point of the file store (for example: `/dev/sda2`).
```

File: specification/nodes/_types/Stats.ts [hunk 6]
```diff
@@ -236,137 +563,382 @@ export class DataPathStats {
...
+   * Current number of open HTTP connections for the node.
+   */
   current_open?: integer
+  /**
+   * Total number of HTTP connections opened for the node.
+   */
   total_opened?: long
+  /**
+   * Information on current and recently-closed HTTP client connections.
+   * Clients that have been closed longer than the `http`.`client_stats`.`closed_channels`.`max_age` setting will not be represented here.
```
Comment (abdonpijpelink) [inline specification/nodes/_types/Stats.ts:644]:
```suggestion
   * Clients that have been closed longer than the `http.client_stats.closed_channels.max_age` setting will not be represented here.
```

**Review by abdonpijpelink — APPROVED** [review]
LGTM. Left a few small suggestions.

File: specification/nodes/_types/Stats.ts (hunk 3 above)
Comment (szabosteve) [reply specification/nodes/_types/Stats.ts]: Great suggestion, thank you!

**Pull request merged**